clang/test/SemaOpenCL/invalid-kernel-attrs.cl

```diff
@@ -39,7 +39,8 @@ kernel __attribute__((intel_reqd_sub_group_size(0))) void kernel15() {} // expec
 
 kernel __attribute__((intel_reqd_sub_group_size(-1))) void kernel16() {} // expected-error {{'intel_reqd_sub_group_size' attribute requires a positive integral compile time constant expression}}
 
-kernel __attribute__((intel_reqd_sub_group_size(8))) __attribute__((intel_reqd_sub_group_size(16))) void kernel17() {} //expected-warning{{attribute 'intel_reqd_sub_group_size' is already applied with different parameters}}
+kernel __attribute__((intel_reqd_sub_group_size(8))) __attribute__((intel_reqd_sub_group_size(16))) void kernel17() {} //expected-warning{{attribute 'intel_reqd_sub_group_size' is already applied with different parameters}} \
```

Comment:
Should that slash be there at end of diagnostic? If not, how come tests pass? If yes, could you please explain why?

Reply:
That's a line continuation character. The effect is as-if the subsequent line were written where the `\` is (it means we don't need to do an `@-1` on that diagnostic to get the correct line ending.

Reply:
Oh! I knew `\` was a line continuation character but didn't ever think of using it with diagnostics this way. Thank you for explaining!